bin/crx.js

```diff
@@ -84,7 +63,16 @@ function keygen (dir, program) {
       throw new Error('key.pem already exists in the given location.');
     }
 
-    generateKeyFile(keyPath);
+    generatePrivateKey()
+      .then(function(privateKey) {
+        fs.writeFile(keyPath, privateKey, function(err){
+          if (err) {
+            throw err;
```

Comment:
This generates an Uncaught exception, doesn't it?

My short experiement in Chrome DevTools console and node 8.9.0:

```javascript
Promise.resolve().then(
	setTimeout(function cb() { throw new Error('File saving failed')}, 10)
).then(
	() => console.log('Promise resolved!'),
    (err) => console.error('Promise rejected')
)

> Promise resolved!
> Uncaught Error: File saving failed
```

The previous code used `resolve` and `reject` and had correct behaviour, I guess.